default aspect collections fail with TypeError

### Version Number

1.19.1

### What branch are you on?

master

### Describe the Bug

```
[2023-09-23 05:49:09,551] [plex_meta_manager.py:819]  [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "//plex_meta_manager.py", line 677, in run_collection                                       |
                                                                 |     builder = CollectionBuilder(config, metadata, mapping_name, collection_attrs, library=library, extra=output_str) |
                                                                 |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ |
                                                                 |   File "/modules/builder.py", line 232, in __init__                                                |
                                                                 |     new_attributes = self.metadata.apply_template(name, self.mapping_name, self.data, self.data[methods["template"]], new_variables) |
                                                                 |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ |
                                                                 |   File "/modules/meta.py", line 476, in apply_template                                             |
                                                                 |     key = key.replace(f"<<{k}>>", v)                                                               |
                                                                 |           ^^^^^^^^^^^^^^^^^^^^^^^^^^                                                               |
                                                                 | TypeError: replace() argument 2 must be str, not ScalarFloat                                       |
                                                                 |                                                                                                    |
[2023-09-23 05:49:09,553] [plex_meta_manager.py:820]  [ERROR]    | Unknown Error: replace() argument 2 must be str, not ScalarFloat                                   |
```
`v` is something like 1.31 here.

Coercing it to a string fixes the issue.


### Relevant Collection/Overlay/Playlist Definition

_No response_

### Logs

_No response_

--- a/modules/meta.py
+++ b/modules/meta.py
@@ -454,10 +454,10 @@ class DataFile:
                         if "<<" in key and ">>" in key:
                             for k, v in variables.items():
                                 if f"<<{k}>>" in key:
-                                    key = key.replace(f"<<{k}>>", v)
+                                    key = key.replace(f"<<{k}>>", f"{v}")
                             for k, v in default.items():
                                 if f"<<{k}>>" in key:
-                                    key = key.replace(f"<<{k}>>", v)
+                                    key = key.replace(f"<<{k}>>", f"{v}")
                             if key not in variables:
                                 variables[key] = value
                     for key, value in variables.copy().items():
@@ -470,10 +470,10 @@ class DataFile:
                         if "<<" in key and ">>" in key:
                             for k, v in variables.items():
                                 if f"<<{k}>>" in key:
-                                    key = key.replace(f"<<{k}>>", v)
+                                    key = key.replace(f"<<{k}>>", f"{v}")
                             for k, v in default.items():
                                 if f"<<{k}>>" in key:
-                                    key = key.replace(f"<<{k}>>", v)
+                                    key = key.replace(f"<<{k}>>", f"{v}")
                         if key not in variables and key not in default:
                             optional.append(key)
 

